Tools/mkdist.py

```diff
@@ -10,7 +10,7 @@ def failed():
     sys.exit(2)
 
 def clean_arg(arg):
-    return arg.replace('`','').replace('\'','').replace('"','').replace(';','').replace('&','').replace(',','').replace('\\','').replace('|','')
+    return arg.replace('`','').replace('\'','').replace('"','').replace(';','').replace('&','').replace(',','').replace('\\','').replace('|','').replace('$','')
```

Comment:
Still bypassable :cry: 
The `(command)` format is possible in `Windows`, leading to `RCE` again